### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Mykyta-snacj**
- [DEMO LINK](https://Mykyta-snacj.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Mykyta-snacj.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by BogdanMaliuta — CHANGES_REQUESTED** [review]
Can`t see demo and tests, fix it please, also remove regions from your css 
image

File: src/index.html [hunk 1]
```diff
@@ -15,8 +15,107 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#header">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="header__logo__img"
```
Comment (Zibi95) [inline src/index.html:38]: According to BEM, an element cannot contain another element directly. Only blocks have elements. Therefore, `header__logo-img` is the correct class name, where `header` is the block and `logo-img `is the element.
```suggestion
            class="header__logo-img"
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+
+  list-style: none;
+}
+
+.nav__list > li:first-child a {
```
Comment (Zibi95) [inline src/style.css:30]: Simplify selectors to adhere to BEM principles and avoid increasing specificity unnecessarily. We can simplify this by using the fact that nav__link have also is-active class.

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
```
Comment (Zibi95) [inline src/style.css:15]: > the header should stretch the full page width (don't use horizontal margin).

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+}
+
+.header__logo__img {
```
Comment (Zibi95) [inline src/style.css:18]: Utilize SCSS nesting with & symbol as it can make the stylesheet more concise and maintainable.

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+
+  --selected-color: #00acdc;
```
Comment (Zibi95) [inline src/style.css:4]: It is a common best practice in modern CSS to define global variables in `:root` to ensure that they are universally accessible and clearly signify their global nature.

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,68 @@
...
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav__link:hover {
```
Comment (Zibi95) [inline src/style.css:54]: Utilzie scss nesting.

**Review by Zibi95 — CHANGES_REQUESTED** [review]
Good job! There few things to fix. Follow suggestions left in comments below.

**Pull request closed without merging**